mantle/kola/harness.go

```diff
@@ -783,11 +783,18 @@ func runProvidedTests(testsBank map[string]*register.Test, patterns []string, mu
 			}
 		}
 
+		if allTestsDenyListed(tests) {
+			if suiteErr == harness.SuiteEmpty {
```

Comment:
or actually it may read better as:

```
if suiteErr == harness.SuiteEmpty && allTestsDenyListed(tests) {
```

Reply:
tested it and works fine (64 chars)

Reply:
> or actually it may read better as:
> 
> ```
> if suiteErr == harness.SuiteEmpty && allTestsDenyListed(tests) {
> ```

only seeing this now, pushed and squashed the new commit